test/smoke/package.json

```diff
@@ -3,13 +3,10 @@
   "version": "0.1.0",
   "main": "./src/main.js",
   "scripts": {
+    "preinstall": "yarn --cwd ../automation",
```

Comment:
Might be better to have `"precompile": "yarn --cwd ../automation compile"`?

Reply:
Actually I intended this to install packages there also. (If packages are already installed, `yarn` is very quick to skip that step.) Otherwise you would have to go manually do a `yarn install` in the `automation` directory first.

Reply:
When you run `yarn` in the root folder it will run for a bunch of sub-folders, can we not hook into that same mechanism? It's being done for `test/smoke` currently.

Reply:
OK, I'll look into it.

Reply:
Done. I added `yarnInstall('test/automation')` to the root `postinstall.js` script. And in this `package.json` I removed the `preinstall` script and updated the `compile` script to also compile the `automation` package. I think that simplifies the workflow when you might be changing both `test/automation` and `test/smoke` at the same time.